local/scripts/install_optional_tools.sh

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env bash
+
+# This script installs optional tools for PATH development:
```

Comment:
I suggest looking for a library/shell script helper that can handle most of this. If one exists, we could remove some  of the code in this directory.

Reply:
Agreed, will add a TODO.